base_location_geonames_import/wizard/geonames_import.py

```diff
@@ -204,31 +187,29 @@ def run_import(self):
         data_file = open(os.path.join(tempdir, '%s.txt' % country_code), 'r',
                          encoding='utf-8')
         data_file.seek(0)
-        logger.info('Starting to create the cities and/or better zip entries')
+        logger.info('Starting to create the cities and/or city zip entries')
         max_import = self.env.context.get('max_import', 0)
         reader = csv.reader(data_file, delimiter='	')
         for i, row in enumerate(reader):
-            res_city = False
-            if self.enforce_cities:
-                res_city = self.create_res_city(row, self.country_id)
-                if res_city in res_cities_to_delete:
-                    res_cities_to_delete -= res_city
-            zip_code = self.create_better_zip(row, self.country_id,
-                                              res_city)
+            res_city = self.create_res_city(row, self.country_id)
+            if res_city in res_cities_to_delete:
```

Comment:
No need to perform this comparison: the substract operation already does that, and it doesn't fail if the element is not in the original set.